### PR TITLE
Added a note where if users did not sign up with Flarum.

### PR DESCRIPTION
**Pull request by rodrigoargumedo**

### CODE REVIEW

File: views/emails/confirm.blade.php [hunk 1]
```diff
@@ -4,3 +4,5 @@
 
 If this was you, simply visit the following link and your account will be activated:
 {{ $url }}
+
+If you did not sign up, just simply ignore this email.
```
Comment (franzliedke) [inline views/emails/confirm.blade.php]: Can you change it to "If you did not sign up, please ignore this email."?

Also, please add a trailing newline.

Comment (rodrigoargumedo) [inline views/emails/confirm.blade.php]: Will do.
On Apr 18, 2015 10:50 AM, Franz Liedke <notifications@github.com> wrote:In views/emails/confirm.blade.php:
> @@ -4,3 +4,5 @@
>  
>  If this was you, simply visit the following link and your account will be activated:
>  {{ $url }}
> +
> +If you did not sign up, just simply ignore this email.

Can you change it to "If you did not sign up, please ignore this email."?

Also, please add a trailing newline.

**Comment by rodrigoargumedo** [comment]
Done in https://github.com/rodrigoargumedo/core/commit/ccc03ddc93b09429efed093e512569a3cac2b954

**Comment by franzliedke** [comment]
Thank you!

**Pull request merged**